containers/base/Dockerfile

```diff
@@ -130,15 +130,16 @@ RUN pip install google-cloud-dataflow==0.4.1
 ADD nbconvert /datalab/nbconvert
 ADD config/ipython.py /etc/ipython/ipython_config.py
 ADD config/nbconvert.py /etc/jupyter/jupyter_notebook_config.py
+# Directory "py" may be empty and in that case it will git clone pydatalab from repo
+ADD py /datalab/lib/pydatalab
```

Comment:
I *think* that if we rename the local directory to "pydatalab", then we can change the ADD instruction to:

    ADD py* /datalab/lib/

And then the base image will still be buildable with just "docker build .." (i.e. the build.sh script is optional unless you want to specify the pydatalib directory).